assets/js/app/battles/components/ListBattle/ListBattle.js

```diff
@@ -0,0 +1,214 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import styled from 'styled-components';
+import { isEmpty } from 'lodash';
+import actions from '../../../../actions';
+import Urls from '../../../../utils/urls';
+
+
+const BattleWrapperStyled = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+`;
+
+const BattleItemStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 300px;
+  height: 300px;
+  margin-bottom: 30px;
+  padding: 20px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid lightgray;
+  box-shadow: 1px 1px 4px lightgray;
+`;
+
+const WinnerBattleStyled = styled.span`
+  color: #009999;
+`;
+
+const HrStyled = styled.hr`
+  width: 100%;
+`;
+
+const OrangePStyled = styled.p`
+  color: #ff6600;
+`;
+
+const DetailLinkStyled = styled(Link)`
+  text-decoration: none;
+  color: #ff6600;
+
+  :hover {
+    color: #ffb84d;
+  }
+`;
+
+function OngoingBattleBox(battle) {
```

Comment:
I do agree with you about how easier it would be to read this file code by moving these _helpers_ components to another folder/file, but I also had a doubt to do that because these _helpers_ components use almost the same styled components.

Considering that, I would have to move the styled components to another file as well. Do you think I really should make these changes to make the code more modular?

Reply:
@arineto  I do agree with you about how easier it would be to read this file code by moving these _helpers_ components to another folder/file, but I also had a doubt to do that because these _helpers_ components use almost the same styled components.

Considering that, I would have to move the styled components to another file as well. Do you think I really should make these changes to make the code more modular?

Reply:
Since this will be more complicated than just moving the components, let's keep this the way you did. Just keep in mind to consider splitting complex code into multiple components in the future.

Another thing is that if you have a simpler component that will be used by many other components you can create a shared folder to put components that will be shared by others and import them.

Reply:
All right, I will. Thank you.